.github/workflows/deploy-previews.yml

```
@@ -249,53 +249,53 @@
             --format='value(status.url)' \
             --platform managed \
             --region ${{ secrets.REGION }})
           echo "url=${service_url}" >>$GITHUB_OUTPUT
 
-  beacon-pr-comment:
-    name: beacon-preview-link-comment
-    if: needs.check.outputs.run_beacon == 'true'
-    needs: [beacon]
+  beacon-api-pr-comment:
+    name: beacon-api-preview-link-comment
+    if: needs.check.outputs.run_beacon_api == 'true'
```

actionlint (1.7.7)

property "check" is not defined in object type {beacon-api: {outputs: {url: string}; result: string}}

(expression)